cocotb/triggers.py

```diff
@@ -407,26 +407,77 @@ def __repr__(self):
 
 
 class RisingEdge(_EdgeBase):
-    """Fires on the rising edge of *signal*, on a transition from ``0`` to ``1``."""
+    """Fires on the rising edge of *signal*, on a transition from ``0`` to ``1``.
+
+    *signal* may have any width, but only transitions from ``0`` to ``1`` are
+    considered a rising edge.
+    """
 
     __slots__ = ()
     _edge_type = 1
 
 
 class FallingEdge(_EdgeBase):
-    """Fires on the falling edge of *signal*, on a transition from ``1`` to ``0``."""
+    """Fires on the falling edge of *signal*, on a transition from ``1`` to ``0``.
+
+    *signal* may have any width, but only transitions from ``1`` to ``0`` are
+    considered a falling edge.
+    """
 
     __slots__ = ()
     _edge_type = 2
 
 
-class Edge(_EdgeBase):
-    """Fires on any value change of *signal*."""
+class ValueChange(_EdgeBase):
+    """
+    Fires on any value change of *signal*.
+
+    *signal* may have any width.
+
+    .. versionadded:: 1.9.0
+    """
 
     __slots__ = ()
     _edge_type = 3
 
 
+class Edge(ValueChange):
+    """Deprecated alias of :class:`cocotb.triggers.ValueChange`.
+
+    .. note::
+
+        ``Edge()`` in cocotb and ``edge`` in SystemVerilog do different things!
+
+        Cocotb's ``Edge()`` trigger considers changes to *all* bits the signal.
+        The SystemVerilog ``edge`` keyword only considers the least significant
+        bit, or in other words, ``edge`` in SystemVerilog is equivalent to
+        ``posedge`` or ``negedge``.
+
+        Hence, the SystemVerilog statement ``@(edge my_multibit_signal)`` is
+        equivalent to
+        ``await First(RisingEdge(my_multibit_signal), FallingEdge(my_multibit_signal))``
+        in cocotb.
+
+    .. deprecated:: 1.9.0
```

Comment:
?
```suggestion
    .. deprecated:: 1.8.2
```

Reply:
I think what @imphil meant was doing a 1.9 release off the stable/1.8 branch which is just deprecations and housekeeping.